### PR TITLE
test: adding e2e test for config volumes EmptyDir and PersistentVolumeClaim

### PR DESCRIPTION
**Pull request by jrangelramos**
# Changes

- :gift: Adding e2e test for PersistenceVolumeClaim and EmptyDir
/kind <kind>

/kind enhancement

### CODE REVIEW

File: test/e2e/scenario_config-volumes_test.go [hunk 1]
```diff
@@ -166,3 +170,134 @@ func TestConfigVolumes(t *testing.T) {
...
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	t.Log("Created test PVC " + pvcName)
+
+}
+
+// tearDownTestPvc removes the PVC used by the test
+func tearDownTestPvc(t *testing.T, pvcName string) {
```
Comment (matejvasek) [inline test/e2e/scenario_config-volumes_test.go:231]: Would it make sense putting this into setupTestPvc() using t.Cleanup()?

**Comment by matejvasek** [comment]
@jrangelramos how big PVC do you request? Maybe it's too big for testing cluster?

**Comment by jrangelramos** [comment]
@matejvasek should be small one.. 1Kb (I believe) .. check it out https://github.com/knative/func/pull/2078/files#diff-4ccb676e582b304e248178f82a6a0e5fcf4e7319a417a9cd16c893cae2b43201R220

**Pull request merged**